packages/frontend/src/ee/pages/AiAgents/ProjectAiAgentEditPage.tsx

```diff
@@ -434,7 +435,45 @@ const ProjectAiAgentEditPage: FC<Props> = ({ isCreateMode = false }) => {
                                         />
                                         <TagsInput
                                             variant="subtle"
-                                            label="Tags"
+                                            label={
+                                                <Group gap="xs">
+                                                    <Text fz="sm" fw={500}>
+                                                        Tags
+                                                    </Text>
+                                                    <HoverCard
+                                                        position="right"
+                                                        withArrow
+                                                    >
+                                                        <HoverCard.Target>
+                                                            <MantineIcon
+                                                                icon={
+                                                                    IconInfoCircle
+                                                                }
+                                                            />
+                                                        </HoverCard.Target>
+                                                        <HoverCard.Dropdown maw="250px">
+                                                            <Text fz="xs">
+                                                                Add tags to
+                                                                control which
+                                                                metrics and
+                                                                dimensions your
+                                                                AI agent can
+                                                                access. See more
+                                                                in our{' '}
+                                                                <Anchor
+                                                                    fz="xs"
+                                                                    c="dimmed"
+                                                                    underline="always"
+                                                                    href="https://docs.lightdash.com/guides/ai-agents#limiting-access-to-specific-explores%2Ffields"
```

Comment:
the url looks a bit off, the `/` in the title is breaking the fragment 
I've updated the title so we could use https://docs.lightdash.com/guides/ai-agents#limiting-access-to-specific-explores-and-fields instead

Reply:
thank you for updating it - I've pushed latest url!